extensions/bombman/standalone.mjs

```diff
@@ -240,6 +243,80 @@ class Standalone {
     }
     return;
   }
+
+  /**
+   * start the game
+   * @param {*} srcExt
+   * @param {String} playerID
+   * @param {Object} kwargs
+   * @param {*} sfInfo
+   * @return {String} - the next state
+   */
+  async s2s_sf_startBombman(srcExt, playerID, kwargs, sfInfo) {
+    const {next} = kwargs;
+    if (this.gameStart) {
+      console.log('game have already started');
```

Comment:
Consider more descriptive logging such as `console.log('[bombman] the game have already started');`